exporter/coralogixexporter/README.md

```diff
@@ -48,40 +42,55 @@ exporters:
     # (Optional) Timeout is the timeout for every attempt to send data to the backend.
     timeout: 30s
 ```
-### Tracing deprecation 
 
-The v0.67 version removed old Jaeger based tracing endpoint in favour of Opentelemetry based one.
+### v0.76.0 Coralogix Domain 
 
-To migrate, please remove the old endpoint field, and change the configuration to `traces.endpoint` using the new Tracing endpoint.
+Since v0.76.0 you can specify Coralogix domain in the configuration file instead of specifying different endpoints for traces, metrics and logs. For example, the configuration below, can be replaced with domain field:
 
 Old configuration:
 ```yaml
 exporters:
   coralogix:
-    # The Coralogix traces ingress endpoint
-    endpoint: "tracing-ingress.coralogix.com:9443"
+    traces:
+      endpoint: "ingress.coralogix.com:443"
+    metrics:
+      endpoint: "ingress.coralogix.com:443"
+    logs:
+      endpoint: "ingress.coralogix.com:443"
 ```
 
-New configuration:
+New configuration with domain field:
 ```yaml
-exporters
+exporters:
   coralogix:
-    # The Coralogix traces ingress endpoint
-    traces:
-      endpoint: "ingress.coralogix.com:443"
+    domain: "coralogix.com"
 ```
 
-### Coralogix's Endpoints 
+### Coralogix's Domain 
 
 Depending on your region, you might need to use a different endpoint. Here are the available Endpoints:
 
-| Region  | Traces Endpoint                          | Metrics Endpoint                     | Logs Endpoint                     |
-|---------|------------------------------------------|------------------------------------- | --------------------------------- |
-| USA1    | `ingress.coralogix.us:443`      | `ingress.coralogix.us:443`      | `ingress.coralogix.us:443`      |
-| APAC1   | `ingress.coralogix.in:443`  | `ingress.coralogix.in:443`      | `ingress.coralogix.in:443`      | 
-| APAC2   | `ingress.coralogixsg.com:443`   | `ingress.coralogixsg.com:443`   | `ingress.coralogixsg.com:443`   |
-| EUROPE1 | `ingress.coralogix.com:443`     | `ingress.coralogix.com:443`     | `ingress.coralogix.com:443`     |
-| EUROPE2 | `ingress.eu2.coralogix.com:443` | `ingress.eu2.coralogix.com:443` | `ingress.eu2.coralogix.com:443` |
+| Region  | Domain                        |
+|---------|---------------------------------|
+| USA1    | `coralogix.us`      |
+| APAC1   | `coralogix.in`      |
+| APAC2   | `coralogixsg.com`   |
+| EUROPE1 | `coralogix.com`     |
+| EUROPE2 | `eu2.coralogix.com` |
```

Comment:
Technically, this list (and the one below as well) includes not just domains but full hostnames. It's just a tad bit confusing (if we use term _domain_ in the typical sense used on the internet).

I wonder if we could find a better term. Region might fit well but could be easily confused with AWS regions. Or maybe simply _hostname_? That would be inclusive of all the entries in the list.